### ISSUE TITLE
ERROR :  input and output paths on network path are unsupported.

### ISSUE THREAD

**Issue by mohitmallick17** [hilts-vaughan/UnlinkMKV-GUI#23]
Hi. I am windows 10(anniversary update) user , 2gb ram, 1 gb graphics.. As UnlinkMKV couldn't give me output I turned to the GUI version. As soon as I open the  unlinkmkv-gui.exe as administrator I get a note at the bottom of the frame that the network path is unsupported. Please provide me a working solution to the problem.
![screenshot 135](https://cloud.githubusercontent.com/assets/22490775/22321625/540478a6-e3bc-11e6-969f-a2ac0958740f.png)

**Comment by hilts-vaughan** [comment]
You can try network paths with network mode. It might work. If you want to
get around the lack of network path support, simply map the network drive
as a local drive on Windows, it'll work.
http://www.laptopmag.com/articles/map-network-drive-windows-10

Does that help?

**Comment by mohitmallick17** [comment]
I don't think it has been helped me.. 
I've created the network path X:/(Which is actually E drive) now when I browse I can't find drive X and still the problem remains same.. I've tried running it from X drive too but no help 
![screenshot 148](https://cloud.githubusercontent.com/assets/22490775/22333786/ccf8ccb2-e3fd-11e6-9312-994a76cf190a.png)

**Comment by hilts-vaughan** [comment]
I have no tried on Windows 10 but will -- however, if it's a network mapped drive you should be able to see it. Have you tried clicking the browse button and just pasting in the path you want directly into the filename textbox and hitting enter?

This sounds like an operating system failure. Would it help if I added support for network paths?

**Comment by mohitmallick17** [comment]
Unfortunately there is no text box in windows  10 to paste the directory.. neither your program allows me to directly paste the directory..

**Comment by mohitmallick17** [comment]
Sure it will.If you added a feature to skip the network path and allow to use local drives everything will be fantastic. Anyways I expect you w'd be able to help me someone

**Comment by hilts-vaughan** [comment]
I guess Windows does not do this. There appears to be some bugs in .Net that sometimes prevents this dialog from working.

https://social.msdn.microsoft.com/Forums/windows/en-US/f0073f80-cedb-4b7a-96b9-20e4b22c2424/folderbrowserdialog-does-not-show-network-drives-on-windows-8?forum=windowscompatibility

You might consider some of the advice there. In the meanwhile, I will roll out a new release perhaps tonight that allows you to paste whatever you want in text box.

I hope that would help. I will also look into adding network drive support, it should be fairly easy at least in native mode.

**Comment by mohitmallick17** [comment]
Yes please it will be a great help.. I don't wanna suggest I am just making a request for the feature that please enable use of local drives directly as tons of Windows 10 users (including me ) face lot of problem in creating network drive I somehow made it but not all can do so I want the new release to allow us the use of our local drives directly like any other application allows us to do.
Second thing as you said, allow us to use of text box directly eveb without browsing and please enable the setenable query as if it throws network path error still we w'd be able to paste the directory.
Thanks a lot for your grateful reply. It really made me happy when you said about publishing a new release. Looking forward to it  😉😊

**Comment by mohitmallick17** [comment]
Dear if you release a newer version can you please let me know ? 
You can reply on comment here.

**Comment by DaniPanadero** [comment]
I've the same problem. Some news?
